engine/src/graphics/device.cpp

```diff
@@ -140,23 +141,27 @@ namespace gen
 		std::vector<vk::DeviceQueueCreateInfo> queueCreateInfos;
 		std::set<u32> const uniqueQueueFamilies = {m_gpu.queueFamily};
 
+		auto enabledFeatures	= vk::PhysicalDeviceFeatures{};
+		auto availableFeatures	= m_gpu.physicalDevice.getFeatures();
+		enabledFeatures.logicOp = availableFeatures.logicOp;
+
 		float const queuePriority = 1.0F;
 		for (u32 const queueFamily : uniqueQueueFamilies)
 		{
 			vk::DeviceQueueCreateInfo const queueCreateInfo({}, queueFamily, 1, &queuePriority);
 			queueCreateInfos.push_back(queueCreateInfo);
 		}
 
-		vk::DeviceCreateInfo const createInfo(
-			{},
-			static_cast<u32>(queueCreateInfos.size()),
-			queueCreateInfos.data(),
-			0,		 // ignored by spec
-			nullptr, // ignored by spec
-			static_cast<u32>(deviceExtensions.size()),
-			deviceExtensions.data(),
-			nullptr // to be used later
-		);
+		auto createInfo = vk::DeviceCreateInfo{};
+
+		createInfo.queueCreateInfoCount	   = static_cast<u32>(queueCreateInfos.size());
+		createInfo.pQueueCreateInfos	   = queueCreateInfos.data();
+		createInfo.enabledExtensionCount   = static_cast<u32>(deviceExtensions.size());
+		createInfo.ppEnabledExtensionNames = deviceExtensions.data();
+		createInfo.pEnabledFeatures		   = &enabledFeatures;
+
+		auto dynamicRenderingFeature = vk::PhysicalDeviceDynamicRenderingFeatures{true}; // NOLINT(readability-implicit-bool-conversion)
+		createInfo.pNext			 = &dynamicRenderingFeature;
```

Comment:
(Note: this is still required, despite the extension being core.)

Reply:
While you're at it, add `vk::PhysicalDeviceSynchronization2FeaturesKHR` to the list as well.